From 0aacbd8dc8203993c136127dce529ddb08a852ee Mon Sep 17 00:00:00 2001
From: Sam Mortimer <sam@mortimer.me.uk>
Date: Thu, 9 Nov 2017 01:41:17 -0800
Subject: [PATCH 134/249] Update aosp tag from android-8.0.0_r3 to
 android-8.0.0_r30

android-8.0.0_r3  is marlin/sailfish OPR6.170623.012, Aug 2017
android-8.0.0_r30 is marlin/sailfish OPR3.170623.013, Nov 2017

Change-Id: I97b199684b9798de1b0f95bd5dc8d6e59c77a20e
---
 default.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/default.xml b/default.xml
index e891dee..ec3b5da 100644
--- a/default.xml
+++ b/default.xml
@@ -11,7 +11,7 @@
   <remote  name="aosp"
            fetch="https://android.googlesource.com"
            review="android-review.googlesource.com"
-           revision="refs/tags/android-8.0.0_r3" />
+           revision="refs/tags/android-8.0.0_r30" />
 
   <default revision="refs/heads/lineage-15.0"
            remote="github"
-- 
2.17.1

